src/aro/Preprocessor.zig

```diff
@@ -1463,6 +1467,17 @@ fn handleBuiltinMacro(pp: *Preprocessor, builtin: RawToken.Id, param_toks: []con
     }
 }
 
+/// Treat whitespace-only paste arguments as empty
+fn getPasteArgs(args: []const Token) []const Token {
+    for (args) |tok| {
+        if (tok.id != .macro_ws) return args;
+    }
+    return &[1]Token{.{
```

Comment:
Is this safe to do if there aren't any runtime dependencies (like the old `tokFromRaw` call)?

Reply:
Yes, it's doing pretty much the same as string literals.